src/loaders/express.js

```diff
@@ -31,12 +31,13 @@ module.exports = (app) => {
         resave: true,
         saveUninitialized: true,
 
-        // Set to true only in prod. Necessary due to reverse proxy setup
+        // Necessary due to reverse proxy setup, but it won't affect anything if there is no proxy.
         // Check https://github.com/expressjs/session/issues/281 for more details
-        proxy: process.env.NODE_ENV === "production",
+        proxy: true,
 
         cookie: {
-            secure: process.env.NODE_ENV === "production",
+            secure: process.env.NODE_ENV !== "test",
+            sameSite: process.env.NODE_ENV === "production" ? "lax" : "none"
```

Comment:
I think we can have `SameSite=Strict` for production environments. Lax does not work when we are sending requests to the back-end cross-origin though, so we would have to have that set as None specifically (it even caused some bugs from what I've heard).

Reply:
I think that will work. Thankfully we have a staging env to test things before production :v